deploy/scripts/src/cri-orange/kbv.ts

```diff
@@ -31,6 +31,8 @@ const loadProfile = selectProfile(profiles)
 const groupMap = {
   kbv: [
     'B01_KBV_01_CoreStubEditUserContinue',
+    'B01_KBV_01_CoreStubEditUserContinue::01_CoreStubCall',
+    'B01_KBV_01_CoreStubEditUserContinue::02_KBVCRICall',
```

Comment:
Adding new values here will mean we need to update the indices elsewhere in the script